### PR TITLE
Fix doc with new javaScript modules API

### PR DESCRIPTION
**Pull request by villander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,16 @@ If you are not using Ember CLI, we recommend using [ember-giftwrap](https://gith
 Interaction with `ember-shepherd` is done entirely through the Tour service, which you can access from any object using the `Ember.inject` syntax:
 
 ```js
-export default Ember.Component.extend({
+import Component from '@ember/component';
+import { inject } from '@ember/service';
```
Comment (rwwagner90) [inline README.md]: This should be `import { inject as service } from '@ember/service';`

File: README.md [hunk 2]
```diff
@@ -328,9 +331,12 @@ A: Don't worry, it's not too bad!  You can just set up an action to start (or ca
 ```js
 // app/routes/application.js
 
-export default Ember.Route.extend({
+import Route from "@ember/routing/route";
+import { inject } from "@ember/service";
```
Comment (rwwagner90) [inline README.md]: Same here

**Comment by villander** [comment]
Fixed @rwwagner90 =D

**Comment by rwwagner90** [comment]
Thanks!

**Pull request merged**